src/pb.rs

```diff
@@ -310,6 +310,13 @@ impl<T: Write> ProgressBar<T> {
         self.add(1)
     }
 
+    /// Resets the start time to now
+    ///
+    /// This allows you to set up the bar some time before it is needed and have the speed and time indications be correct
```

Comment:
Method name is pretty clear. Let's remove the rest of the comment. 

```suggestion
    /// Resets the start time to now
```

Reply:
Done